modules/swagger-codegen/src/main/resources/ruby/gemspec.mustache

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |s|
   s.summary     = "{{gemSummary}}{{^gemSummary}}{{{appName}}} Ruby Gem{{/gemSummary}}"
   s.description = "{{gemDescription}}{{^gemDescription}}{{{appDescription}}}{{^appDescription}}{{{appName}}} Ruby Gem{{/appDescription}}{{/gemDescription}}"
   s.license     = "{{gemLicense}}{{^gemLicense}}{{{licenseInfo}}}{{^licenseInfo}}Apache 2.0{{/licenseInfo}}{{/gemLicense}}"
+  s.required_ruby_version = "{{gemRequiredRubyVersion}}{{^gemRequiredRubyVersion}}>= 1.9{{/gemRequiredRubyVersion}}"
```

Comment:
@wing328 Thanks for the pointer, done in https://github.com/swagger-api/swagger-codegen/pull/3718/commits/b68b53975340f945d291f27fa75f70b4781aae8f following gemLicense pattern.